src/react_native/core.cljs

```diff
@@ -6,6 +6,7 @@
             [react-native.flat-list :as flat-list]
             [react-native.platform :as platform]
             [react-native.section-list :as section-list]
+            ["react-native-draggable-flatlist" :default DraggableFlatList]
```

Comment:
better to have a separate `react_native/draggable_flatlist.cljs` namespace

Reply:
Removed, no need anymore